soroban-env-host/src/test/hostile.rs

```diff
@@ -773,22 +772,13 @@ fn test_corrupt_custom_section() -> Result<(), HostError> {
     ));
 
     let ledger_protocol = meta::get_ledger_protocol_version(meta::INTERFACE_VERSION);
-    let ledger_pre = meta::get_ledger_protocol_version(meta::INTERFACE_VERSION);
-
-    let interface_meta = |proto: u32, pre: u32| -> Vec<u8> {
-        let iv = (proto as u64) << 32 | pre as u64;
-        let entry = ScEnvMetaEntry::ScEnvMetaKindInterfaceVersion(iv);
-        let bytes = Vec::<u8>::new();
-        let mut w = Limited::new(bytes, Limits::none());
-        entry.write_xdr(&mut w).unwrap();
-        w.inner
-    };
+    let ledger_pre = meta::get_pre_release_version(meta::INTERFACE_VERSION);
```

Comment:
This is interesting -- presumably this is why the test changed, was it a bug?

Reply:
Yes, this was a bug in the test. We were passing the protocol version number as the prerelease number. The test passed because it still satisfied the condition: input mismatched with the expected prerelease number. 
Yes, this is why the test changed.